mivs/configspec.ini

```diff
@@ -61,11 +61,17 @@ waitlisted = string(default="waitlisted")
 declined = string(default="declined")
 banned = string(default="banned")
 
```

Comment:
My original implementation didn't account for the 2-phase review process, wherein judges first review the game videos and then in phase 2 they review the games themselves.  I've since split these out into separate database columns and separate enum sections in the config.